x-pack/plugins/ml/public/application/components/data_grid/feature_importance/use_classification_path_data.tsx

```diff
@@ -165,9 +181,85 @@ export const buildClassificationDecisionPathData = ({
     }
     return undefined;
   });
+
+  // get the baseline for the current class from the trained_models metadata
+  const baselineClass = baselines.find(
+    (bl) => getStringBasedClassName(bl.class_name) === currentClass
+  );
   const filteredFeatureImportance = mappedFeatureImportance.filter(
     (f) => f !== undefined
   ) as ExtendedFeatureImportance[];
 
-  return buildDecisionPathData(filteredFeatureImportance);
+  const finalResult: DecisionPathPlotData = filteredFeatureImportance
+    // sort so absolute importance so it goes from bottom (baseline) to top
```

Comment:
Nit - typo? `Sort by absolute importance...` ?

Reply:
Thanks for catching that. Updated here: https://github.com/elastic/kibana/pull/80229/commits/8121fe6f471872d24c1b1f6367ad11c1cf4fe481